src/components/codeblocks/CodeblockTree.svelte

```diff
@@ -8,10 +8,19 @@
 	import FlatEdgeList from "../FlatEdgeList.svelte";
 	import NestedEdgeList from "../NestedEdgeList.svelte";
 	import CodeblockErrors from "./CodeblockErrors.svelte";
+	import { onMount } from "svelte";
 
 	export let plugin: BreadcrumbsPlugin;
 	export let options: ICodeblock["Options"];
 	export let errors: BreadcrumbsError[];
+	export let file_path: string;
+
+	let all_paths = [];
+
+	// if the file_path is an empty string, so the code block is not rendered inside note, we fall back to the active file store
+	$: active_file_path = file_path ? file_path : ($active_file_store ? $active_file_store.path : undefined);
```

Comment:
Same here, we don't need the intermediate variable or fallbacks. `file_path` is always a string, and empty is fine